DetectorDescription/DDCMS/interface/DDFilteredView.h

```diff
@@ -179,6 +179,21 @@ namespace cms {
       return isA<dd4hep::Tube>(std::forward<Ts>(ts)...);
     }
 
+    template <typename... Ts>
+    auto isAPolycone(Ts&&... ts) const -> decltype(isA<dd4hep::Polycone>(std::forward<Ts>(ts)...)) {
```

Comment:
Touristic question here, didn't dig too much:
decltype(isA<dd4hep::Polycone>(std::forward<Ts>(ts)...))
Is that not... always a boolean ?

Reply:
template <typename... Ts>
bool isAPolycone(Ts&&... ts) const {
      return isA<dd4hep::Polycone>(std::forward<Ts>(ts)...);
}

Reply:
Not sure this needs the template:
```
bool isAPolycone(dd4hep::Solid solid)  {
  return dd4hep::isA<dd4hep::Polycone>(solid);
}
```
Should match all possible shapes as well.